chainreg/chainregistry.go

```diff
@@ -380,6 +390,8 @@ func NewPartialChainControl(cfg *Config) (*PartialChainControl, func(), error) {
 			DisableTLS:           true,
 			HTTPPostMode:         true,
 		}
+		cfg.RPCConfig = *rpcConfig
```

Comment:
I don't think we need any of this. Since the outbound peer check isn't its own check (e.g. it doesn't get its own configuration level flags and is using the chain backend config variables), the whole logic should be done within `cc.HealthCheck`. Especially, since we're currently only log and don't return any errors.